classes/classes/BodyParts/LowerBody.as

```diff
@@ -0,0 +1,385 @@
+/**
+ * Created by aimozg on 25.04.2017.
+ */
+package classes.BodyParts {
+import classes.Creature;
+import classes.internals.Utils;
+
+public class LowerBody extends BodyPart{
+	public var legCount:int = 2;
+
+	override public function set type(value:int):void {
+		super.type = value;
+		// Reset leg count
+		switch(value) {
+			case LOWER_BODY_TYPE_GOO:
+			case LOWER_BODY_TYPE_NAGA:
+				legCount = 1;
+				break;
+			case LOWER_BODY_TYPE_BEE:
+			case LOWER_BODY_TYPE_BUNNY:
+			case LOWER_BODY_TYPE_CAT:
+			case LOWER_BODY_TYPE_CHITINOUS_SPIDER_LEGS:
+			case LOWER_BODY_TYPE_DEMONIC_CLAWS:
+			case LOWER_BODY_TYPE_DEMONIC_HIGH_HEELS:
+			case LOWER_BODY_TYPE_DOG:
+			case LOWER_BODY_TYPE_DRAGON:
+			case LOWER_BODY_TYPE_ECHIDNA:
+			case LOWER_BODY_TYPE_FERRET:
+			case LOWER_BODY_TYPE_FOX:
+			case LOWER_BODY_TYPE_HARPY:
+			case LOWER_BODY_TYPE_HUMAN:
+			case LOWER_BODY_TYPE_KANGAROO:
+			case LOWER_BODY_TYPE_LIZARD:
+			case LOWER_BODY_TYPE_RACCOON:
+			case LOWER_BODY_TYPE_SALAMANDER:
+				legCount = 2;
+				break;
```

Comment:
No specific reason. Actually, I've considered grouping content from `defaultLegCount`, `foot`, `feet`, `leg`, `legs` by enum value and not by usage (similar to `createMapFromPairs` in Appearance), and this way it "screams louder"